### ISSUE TITLE
PSX Bios Cannot Be Loaded on Switch Lite

### ISSUE THREAD

**Issue by reianayami** [libretro/RetroArch#15267]
When in all version of retroarch tested (1.12, 1.9.11, 1.14, 1.15) all PSX games report 'bios could not be loaded, add for better compatibility'. The bios IS there and is seen by the core information, so it must be related to retroarch itself. Same exact files when on a normal v1 switch work without issue with no bios message.

Here is a log of it not running. Dont have a screen shot but bios IS FOUND in the core menu, and named correctly.

https://pastebin.com/rGS0mM79

**Comment by bslenul** [comment]
If core information says "present" but the core doesn't find it, usually that means you have `Settings > Saving > System Files are in Content Directory` enabled. So either disable that, or put the BIOS in the same folder as the game.

**Comment by reianayami** [comment]
Yes I have tried all of that, pointing to a bios folder that has only those bios, resetting and using the default location, it’s none of those. 

Is there any game that simply doesn’t work without bios so I can test if it’s a false positive?

**Comment by bslenul** [comment]
MGS I believe just gives a black screen on boot if you don't have a BIOS.

**Comment by reianayami** [comment]
> MGS I believe just gives a black screen on boot if you don't have a BIOS.

It def doesn’t see it then I get a black screen when booting it. So it’s def a problem

like I said I’ll try the same config on a OLED when I get back from vacation to see if it persists but it’s def currently only when on the lite. 

Hopefully it can get fixed somehow. Do you know if there’s a specific line in the config to force it to use a specific bios? Like instead of a “bios directory” a line to force a specific one?

**Comment by reianayami** [comment]
Just giving one more bit of information, other bios files such as Sega CD work without issue. So it seems specific to psx
